daemon/create_unix.go

```diff
@@ -117,8 +116,11 @@ func (daemon *Daemon) populateVolume(ctx context.Context, c *container.Container
 		log.G(ctx).WithError(err).Debugf("can't copy data from %s:%s, to %s", c.ID, mnt.Destination, volumePath)
 		return errors.Wrapf(err, "failed to populate volume")
 	}
-	defer mnt.Cleanup(compatcontext.WithoutCancel(ctx))
-	defer cleanup(compatcontext.WithoutCancel(ctx))
+	defer func() {
+		ctx := context.WithoutCancel(ctx)
+		cleanup(ctx)
+		mnt.Cleanup(ctx)
+	}
```

Comment:
```suggestion
	}()
```
defer needs a function call expression

Reply:
DOH!! I thought of that when writing.. and then didn't 🙈

Reply:
Fixed 😄